src/components/ProductImage/ProductImage.js

```diff
@@ -51,9 +54,13 @@ class ProductImage extends Component {
 
     return (
       <div className={cx(`${CN}__container`)}>
-        <figure className={`${CN}__big-image-container`} onClick={() => this.clickHandler(this.renderBigImage())}>
-          <img src={this.renderBigImage()} className={CN} />
-        </figure>
+        <div className={`${CN}__big-image-container`} onClick={() => this.clickHandler(this.renderBigImage())}>
```

Comment:
That's really bad idea to use such kind of functions inside render. What is suppose to do? Can it be separate component's method?